kata-webhook/deploy/webhook.yaml

```diff
@@ -20,8 +20,11 @@ spec:
     spec:
       containers:
         - name: pod-annotate-webhook
-          image: fuentess/kata-webhook:20200707
+          image: katadocker/kata-webhook-example:latest
```

Comment:
Is there any reason this shouldn't be used in a production environment or any reason why someone should need to create their own Kata webhook? If it can be used in production I would consider removing the "-example" from the name.

Reply:
I think it could be run in production, but the behavior of it should depend on the use case. Maybe someone would want to add additional changes to the spec or consider different scenarios where they do not want to use kata. Currently we have only added conditions that make sense for our CI.